src/trix/elements/trix_editor_element.coffee

```diff
@@ -113,6 +113,23 @@ Trix.registerElement "trix-editor", do ->
         @setAttribute("trix-id", ++id)
         @trixId
 
+  labels:
+    get: ->
+      if @inputElement?.hasAttribute("id")
+        inputId = @inputElement.getAttribute("id")
+
+        inputLabels = @ownerDocument?.querySelectorAll("label[for=\"#{inputId}\"]")
+
+      if @hasAttribute("id")
+        trixEditorId = @getAttribute("id")
+
+        trixEditorLabels = @ownerDocument?.querySelectorAll("label[for=\"#{trixEditorId}\"]")
+
+      if findClosestElementFromNode(@parentElement, "label")
```

Comment:
In response to [a CI failure in IE11](https://travis-ci.org/github/basecamp/trix/builds/725973263#L345-L351) related to missing out on `contains`, I tried to call `findClosestEleementFromNode`, but in tests locally this resolved the the `<trix-editor>` element instead of the `<label>`. That surprised me, and going up one level in the hierarchy seemed to resolve the issue.